### PR TITLE
fix: not include everything for rspack

### PR DESCRIPTION
**Pull request by bvanjoi**
Now we're constrained to loading only those that come from unplugin

### CODE REVIEW

**Comment by antfu** [comment]
Do you know when rspack 0.4.0 will come out? And would the new changes still work in 0.3.0?

**Comment by bvanjoi** [comment]
> Do you know when rspack 0.4.0 will come out

It was released last week, and version 0.4.1 is set to be released this week.

> And would the new changes still work in 0.3.0?

No, the changes necessitate a minimum version of 0.4.1.

**Comment by bvanjoi** [comment]
It's peculiar that some CI checks have passed while others have failed. :|

File: package.json [hunk 1]
```diff
@@ -44,8 +44,8 @@
     "@ampproject/remapping": "^2.2.1",
     "@antfu/eslint-config": "^1.0.0-beta.29",
     "@antfu/ni": "^0.21.8",
-    "@rspack/cli": "^0.3.8",
-    "@rspack/core": "^0.3.8",
+    "@rspack/cli": "0.4.0-canary-a9e7d87-20231127152134",
+    "@rspack/core": "0.4.0-canary-a9e7d87-20231127152134",
```
Comment (9aoy) [inline package.json:48]: 0.4.1 is out, we can update it!

**Comment by bvanjoi** [comment]
The latest patch includes changes:

- use `@rspack/core@0.4.1`
- enable `--no-threads` for vitest because I suspect the previous panic was caused by multiple threads, so I have disabled them.

**Comment by ClarkXia** [comment]
Any progress?

**Comment by Jungzl** [comment]
> Any progress?

You can use this [patch](https://gist.github.com/Jungzl/bbe905b2c499cc4aae7808f2091c8e56) as a workaround.

**Comment by bvanjoi** [comment]
friendly ping @antfu , can we proceed with this test?

**Pull request merged**